docs/operations/licensing.md

```diff
@@ -78,55 +78,73 @@ Ondat cluster by browsing the __Licence__ page as shown below:
+      app: storageos-cli
+      run: cli
+  template:
+    metadata:
+      labels:
+        app: storageos-cli
+        run: cli
+    spec:
+      containers:
+      - command:
+        - /bin/sh
+        - -c
+        - "while true; do sleep 3600; done"
+        env:
+        - name: STORAGEOS_ENDPOINTS
+          value: http://storageos:5705
+        - name: STORAGEOS_USERNAME
+          value: storageos
+        - name: STORAGEOS_PASSWORD
+          value: storageos
+        image: storageos/cli:v2.5.0
+        name: cli
+END
+```
 
-Contact Ondat Customer Success team via getstarted@ondat.io and provide the
-following information:
-- **First** and **last** name
-- **Company** **name** and **role** (optional for Personal licence, mandatory
-  for Commercial licence)
-- **Cluster ID**
+> ⚠️ Be sure to edit the environment variables appropriately for your target cluster, eg. the
+> username/password for the administrative user. 
 
-In return to the registration, you will receive a licence file with a content
-similar to this one:
+Once the pod is launched, you can retrieve it's unique identifier:
 
-```
-clusterCapacityGiB: 5120 
-clusterID: 164237eb-f88a-4bb8-a7cf-a23d468e07c0 
-customerName: storageos
-expiresAt: "2021-11-15T14:00:00Z" 
-features:
-- nfs
-kind: project
 
-------------- LICENCE SIGNATURE -------------
-KyjNleTcdmieZVLmZ/rg0SzdAM7I/CH0j22FIFJJSJaeB71OvQrTMtHGyL5TSFNMrEGbyh1HQlDgZb5A
-V1HyjBlS3LjoB/MoagulTxIlZh/R8eRXCOQ46qNZ8Yb7+dHLdCVXBnRqZT11hLqZsMqIeO1y9f5dw65H
-kvl6vWW7YIS9r655S25jMMU7brrGDQVdjvU7tSA74BrnzDFHu7/poopIuFqcxZc/NLrKp/akkvyZI5Ex
-1wH7D4onjVG2pgi30Kia+mjbI1B9pxQyRppQQ4hNXy4qBUUNMFh0menh0wHdQoM1VLU4Il22PrkeICV0
-NaalLsK/96bJov6tpbg96g==
+```shell
+POD=$(kubectl -n storageos get pod -ocustom-columns=_:.metadata.name --no-headers -lapp=storageos-cli)
 ```
 
-## Ondat GUI - Applying a Licence via the Ondat GUI To apply a licence:
-1.  Browse the __Licence__ page of the Ondat GUI
-1.  Click the __Upgrade__ button, for the specific licence level you purchased
-1.  Paste the licence key into the pop-up window
-1.  To apply the license click __Upgrade__
-
-> ⚠️ It is crucial to paste all the licence text into the pop-up and not
-> just the signature. Keep in mind that the encoding of the file must not
-> change.
-
-![Apply Licence Key](/images/docs/operations/licensing/apply-licence-key.png)
+Now that the `POD` variable is set, the pod is accessible via that variable for the rest of the lifetime of
+that terminal. If you open a new terminal, you'll need to run this command again to rediscover the ID of
+the pod.
 
 ## Ondat CLI - Retrieving a cluster ID via the Ondat CLI
 
-For the purpose of this document, it is assumed that the Ondat CLI has been deployed locally on
-the same machine hosting kubectl. To retrieve the cluster ID using the Ondat
-CLI, perform the following:
+> ⚠️ Make sure that you have the CLI running, as described in the previous step!
+
+```shell
+kubectl -n storageos exec $POD -- storageos get volumes
```

Comment:
This should storageos get licence or get cluster, no?

Reply:
Great spot! Updated, also replaced the current output with new-format output from those commands.